lib/Steps/Integrations/NextJs.ts

```diff
@@ -97,6 +94,83 @@ export class NextJs extends BaseIntegration {
     return this.shouldConfigure;
   }
 
+  private async _createSentryCliConfig(
+    cliProps: SentryCliProps,
+  ): Promise<void> {
+    const cliPropsToWrite = { ...cliProps };
+    const authToken = cliPropsToWrite['auth/token'];
+
+    /**
+     * To not commit the auth token to the VCS, instead of adding it to the
+     * properties file (like the rest of props), it's added to the Sentry CLI
+     * config, which is added to the gitignore. This way makes the properties
+     * file safe to commit without exposing any auth tokens.
+     */
+    if (authToken) {
```

Comment:
Can we log an error if auth token is undefined (else condition on this if statement)?